apps/CricMyBrainAndroidApp/app/src/main/java/com/cricmybrain/cricmybrainandroidapp/LocalServerActivity.java

```diff
@@ -0,0 +1,39 @@
+package com.cricmybrain.cricmybrainandroidapp;
+
+import android.app.Activity;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.KeyEvent;
+import android.webkit.WebView;
+
+public class LocalServerActivity extends Activity {
+    WebView myWebView;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_local_server);
+
+        myWebView = (WebView) findViewById(R.id.localServerWebview);
+        myWebView.setWebViewClient(new MyWebViewClient(LocalServerActivity.this));
+        myWebView.getSettings().setBuiltInZoomControls(true);
+        myWebView.getSettings().setSupportZoom(true);
+        myWebView.getSettings().setJavaScriptCanOpenWindowsAutomatically(true);
+        myWebView.getSettings().setJavaScriptEnabled(true);
+        myWebView.getSettings().setAllowFileAccess(true);
+        myWebView.getSettings().setDomStorageEnabled(true);
+        myWebView.loadUrl("http://10.0.0.117");
```

Comment:
You can use Debug vs Release mode for local server testing.

In Debug Mode ,
if (BuildConfig.DEBUG) {
  // we show a text box in MainActivity
  // and in the callback, pass string as Argument to Server Activity
}

Thus in Debug Mode we can connect to any server, LocalServerActivity is dupe of RemoteServer and just URL is different. We do not need LocalServer Activity at all